daemon/execdriver/windows/run.go

```diff
@@ -228,14 +172,23 @@ func (d *Driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	logrus.Debugf("CommandLine: %s", createProcessParms.CommandLine)
 
 	// Start the command running in the container.
-	var pid uint32
-	pid, err = hcsshim.CreateProcessInComputeSystem(c.ID, createProcessParms)
-
+	pid, stdin, stdout, stderr, err := hcsshim.CreateProcessInComputeSystem(c.ID, pipes.Stdin != nil, true, !c.ProcessConfig.Tty, createProcessParms)
 	if err != nil {
 		logrus.Errorf("CreateProcessInComputeSystem() failed %s", err)
 		return execdriver.ExitStatus{ExitCode: -1}, err
 	}
 
+	// Now that the process has been launched, begin copying data to and from the named pipes for the std handles.
+	if stdin != nil {
```

Comment:
At least a function that does `setupPipes` like in linux would be welcome, that should probably be the same for run and exec, no?